models/public/erfnet/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This is a ONNX* version of `ERFNet` model designed to perform real-time lane detection.
+  This model is pre-trained in PyTorch* framework and retrained by CULane.
+  For details see repository <https://github.com/Zhangxianwen2021/ERFNet>,
+  paper of ERFNet<https://doi.org/10.1109/TITS.2017.2750080>and repository <https://github.com/cardwing/Codes-for-Lane-Detection/tree/master/ERFNet-CULane-PyTorch>
+task_type: instance_segmentation
```

Comment:
it is not instance segmentation model, jugging by output, model performs semantic_segmentation